src/RoleObjects/TreeItemData.js

```diff
@@ -85,6 +87,7 @@ export default class TreeItemData extends ListItemData {
    * Keydown listener for an tree item
    * @access private
    */
+
```

Comment:
Nit: please remove the blank line. A function comment should be right up against its associated function.

Reply:
Done